test/nodes/test_guardrails.py

```
@@ -0,0 +1,938 @@
+        mock_iglobal = types.SimpleNamespace(engine=engine, config={})
+        inst.IGlobal = mock_iglobal
+
+        forwarded = []
+        inst.instance = types.SimpleNamespace(writeQuestions=lambda q: forwarded.append(q))
+
+        q = FakeQuestion(questions=[FakeQuestionText('What is the weather?')])
+        inst.writeQuestions(q)
+        assert len(forwarded) == 1
+
+    def test_write_questions_blocks_injection(self):
+        IInstance, EngineClass, ViolationClass, FakeQuestion, FakeQuestionText, _ = self._load_iinstance_class()
```

Ruff (0.15.7)

[warning] Unpacked variable `ViolationClass` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)